src/Sylius/Bundle/ApiBundle/Doctrine/ORM/QueryExtension/Admin/Promotion/PromotionCouponExtension.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Doctrine\ORM\QueryExtension\Admin\Promotion;
```

Comment:
```suggestion
namespace Sylius\Bundle\ApiBundle\Doctrine\ORM\QueryExtension\Admin\PromotionCoupon;
```
or
```suggestion
namespace Sylius\Bundle\ApiBundle\Doctrine\ORM\QueryExtension\Admin\Promotion\PromotionCoupon;
```
🤔

Reply:
I did it intentionally, as the coupon now depends on the promotion, but I’m fine with moving it to a dedicated directory